lagom-cloud-object-storage-example/account-impl/src/main/resources/cloud-object-storage.conf.template

```diff
@@ -0,0 +1,11 @@
+cloud-object-storage {
+
+  host = ""
+  bucket = ""
+  
+  credentials {
+    access-key-id = ""
+    secret-access-key = ""
+  }
+
+}
```

Comment:
Could we make this into `cloud-object-storage-reference.conf`?

Reply:
We may need to add comments here (at least a link on how to obtain the values and/or fake values to be used as examples).

Reply:
This is not yet finished. I plan to document it in the README file. I can add comments here as well.